### ISSUE TITLE
Update lodash for security advisory

### ISSUE THREAD

**Issue by heddn** [glebmachine/postcss-easysprites#183]
```
  Moderate        Sandbox Breakout / Arbitrary Code Execution                   
                                                                                
  Package         static-eval                                                   
                                                                                
  Patched in      >=2.0.0                                                       
                                                                                
  Dependency of   postcss-easysprites [dev]                                     
                                                                                
  Path            postcss-easysprites > spritesmith > pixelsmith >              
                  ndarray-fill > cwise > static-module > static-eval            
                                                                                
  More info       https://nodesecurity.io/advisories/548                        
                                                                                
                                                                                
  Low             Prototype Pollution                                           
                                                                                
  Package         lodash                                                        
                                                                                
  Patched in      >=4.17.5                                                      
                                                                                
  Dependency of   postcss-easysprites [dev]                                     
                                                                                
  Path            postcss-easysprites > lodash                                  
                                                                                
  More info       https://nodesecurity.io/advisories/577           
```

**Comment by glebmachine** [comment]
@heddn Thanks!

Could you please update dependencies and post PR, i will merge it asap

**Comment by heddn** [comment]
@glebmachine I've tried. But ran into issues. Perhaps you can take over?

**Comment by patrickcate** [comment]
@heddn this is partially fixed with the latest [0.1.10 release](https://github.com/glebmachine/postcss-easysprites/releases/tag/0.1.10). The remaining security alert for Gulp will be fixed with #191
